conf.d/health_alarm_notify.conf

```diff
@@ -275,6 +278,26 @@ SLACK_WEBHOOK_URL=""
 DEFAULT_RECIPIENT_SLACK=""
 
 
+#------------------------------------------------------------------------------
+# alerta (alerta.io) global notification options
+
+# multiple recipients can be given like this:
+#                  "Environment1 Environment1 ..."
+
+# enable/disable sending alerta notifications
+SEND_ALERTA="YES"
+
+# set your alerta server API url
+ALERTA_WEBHOOK_URL=""
+
+# Login to you arerta server and create an API KEY with write permisions
+ALERTA_API_KEY=""
+
+# if a role's recipients are not configured, a notification will be send to
+# this Environment (empty = do not send a notification for unconfigured roles):
+DEFAULT_RECIPIENT_ALERTA=""
+
+
```

Comment:
At the end of this file you need to add entries like these:

```
role_recipients_alerta[sysadmin]="${DEFAULT_RECIPIENT_ALERTA}"
```

These are used to support role based recipients.